src/poetry/console/application.py

```diff
@@ -47,6 +50,17 @@ def _load() -> Command:
     return _load
 
 
+@contextlib.contextmanager
+def switch_working_directory(path: Path) -> Iterator[Path]:
```

Comment:
There is already `poetry.utils.helpers:directory` for that. However, I would consider changing the name of the helper to `chdir` or to your function name to make it more meaningful.

Reply:
Good catch :+1: